R/timeseries_plot.R

```diff
@@ -1,25 +1,25 @@
-#' Produce an timeseries graph
+#' Produce a timeseries graph
 #' 
 #' @description Creates a timeseries graph for each element and station given. Takes a data frame as an input and the relevant columns to create the plot.
 #' Creates a graph using \code{ggplot2} and returns a timeseries plot.
 #'
-#' @param data The data.frame to calculate from.
-#' @param date_time The name of the date column in \code{data}.
-#' @param elements The name of the column in \code{data} to apply the function to.
-#' @param station The name of the station column in \code{data}, if the data are for multiple station.
+#' @param data `[data frame]` The data.frame to calculate from.
+#' @param date_time [\code{\link[base]{Date}}] The name of the date column in \code{data}.
```

Comment:
@dannyparsons I assume you mean something like this?

Reply:
Yes, looks good. Although I'm not sure we need the `\link` part? Unless they do it in checkmate too?

Reply:
Ok. I don't mind either way so feel free to leave it out if it doesn't work.